docs/core-concepts/bot-sequences/actions.mdx

```diff
@@ -1,9 +1,624 @@
+clicking on certain icons or objects that may have dynamic positions or are changing often during development.
+
+**Data Format**
+
+`type` is set to `Mouse_CVImage`.
+
+- `imageData` - The image data in one of the following formats...
+    - The base64 encoded string of the JPG image data - This must be the entire JPG image file, not just the visible bytes.
+    - A file:// path to a JPG or PNG image - Be careful when using relative paths as these will be interpreted differently in the Editor vs Runtime builds.
+    - A resource:// path to a READABLE Texture2D in one of your project's Resources folders.
+    - A resource:// path to a .bytes TextAsset in one of your project's Resources folders that is a JPG or PNG file saved with a .bytes file extension.  This can be used if you do not want Unity to import your image as a Texture.
+- `withinRect` - An optional (can be null/undefined) field to limit the search area to a specific pixel region of the current frame.  The SDK will linearly tranform the supplied `rect` to fit the current resolution using the `screenSize` as the initial reference resolution.
+    - `screenSize` - The reference resolution in pixels which defines the screen space that `rect` is defined within.
+    - `rect` - The position (x=0, y=0 is bottom left) and size (width, height) of the rectangle that must contain the supplied image data.  The values are defined in pixels.
+- `actions` - The list of mouse actions to take at the center point of the returned `rect`.
+
+**Notes**
+
+The `CVImage` type is still in an experimental phase and may provide inconsistent or unexpected results in many situations.
+- Multiple matches of the specified image within a frame may provide incorrect or inconsistent result bounds.
+- False positives or incorrect result bounds may occur if pixel regions similar to the specified image exist within a frame.
+
+**How to get the data for the `imageData` field as a base64 encoded string**
+
+1. Use a screenshot tool to capture the target area of your game and save this as a JPG file (PNG/BMP/etc are NOT supported at this time).
+2. Run one of the following commands to get the base64 encoded string of the image using the scripts in [this directory](https://github.com/Regression-Games/RGUnityBots/tree/main/user-tools).
+
+```shell
+CURRENT_PATH> python encode_jpg_base64.py sample_images/tree.jpg
+/9j/4AAQ...<rest of the base64 encoded image data>...VLj3ieS/9k=
+```
+or
+```shell
+CURRENT_PATH> ./encode_jpg_base64.sh sample_images/tree.jpg
+/9j/4AAQ...<rest of the base64 encoded image data>...VLj3ieS/9k=
+```
+
+**Example: Using base64 byte[]**
+```json
+{
+    "name": "CV Image Action: Click Menu Change Profile Button",
+    "description": "Moves the mouse over the change profile button, then clicks and releases on the button. Criteria waits for the action to complete.",
+    "endCriteria": [
+        {"type":"ActionComplete", "transient":false, "data":{}}
```

Comment:
This is the first time `endCriteria` has been filled in.  Either exclude this everywhere (Eesh) or fill this in everywhere and provide a quick explainer at the start.

Reply:
Good catch, forgot to remove during the copy paste. Fixed!